ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Homework
+{
+    using System;
+    using System.IO;
+    using System.Collections.Generic;
+    using System.Text;
+    using System.Formats;
+    using System.Text.RegularExpressions;
+
+    public static class MyExtensions
+    {
+        public static bool IsWeekend(this DateTime date)
+        {
+            if (date.DayOfWeek == DayOfWeek.Sunday || date.DayOfWeek == DayOfWeek.Saturday) return true;
```

Comment:
this is very bad style to do if (...) return true; return false :)
instead return condition, it's already bool)